modules/local/bin_summary.nf

```diff
@@ -18,18 +17,17 @@ process BIN_SUMMARY {
     path "versions.yml"    , emit: versions
 
     script:
-    def busco_summary  = busco_sum.sort().size() > 0 ?  "--busco_summary ${busco_sum}" : ""
-    def checkm_summary = checkm_sum.sort().size() > 0 ?  "--checkm_summary ${checkm_sum}" : ""
+    def binqc_summary  = binqc_sum.sort().size() > 0 ?  "--binqc_summary ${binqc_sum}" : ""
     def quast_summary  = quast_sum.sort().size() > 0 ?  "--quast_summary ${quast_sum}" : ""
     def gtdbtk_summary = gtdbtk_sum.sort().size() > 0 ? "--gtdbtk_summary ${gtdbtk_sum}" : ""
     def cat_summary    = cat_sum.sort().size() > 0 ?    "--cat_summary ${cat_sum}" : ""
     """
     combine_tables.py --depths_summary ${bin_depths} \
-                    $busco_summary \
-                    $checkm_summary \
+                    $binqc_summary \
                     $quast_summary \
                     $gtdbtk_summary \
                     $cat_summary \
+                    --binqc_tool ${params.binqc_tool} \
```

Comment:
To try and conform to nf-core module standards as much as possible, can you instead pass this via an input channel as it's presumably a mandatory option?